src/attribute_O2_3.c

```diff
@@ -19,3 +19,116 @@ u16 sub_08018EE8(u8 idx)
 {
     return gUnk_08B80178[idx];
 }
+
+// I am not sure how to fix the rest issues.
```

Comment:
If you really have no idea, then send all the links to me. I don't want to scan all your functions here.

Reply:
Sorry. I will send the links.